src/ai/backend/manager/api/schema.graphql

```diff
@@ -43,7 +43,13 @@
     reference: String
     architecture: String = "x86_64"
   ): Image
-  images(is_installed: Boolean, is_operation: Boolean): [Image]
+  images(
+    is_installed: Boolean @deprecated(reason: "Deprecated since 24.03.4.")
+    is_operation: Boolean @deprecated(reason: "Deprecated since 24.03.4.")
+
+    """Added in 24.03.4."""
+    image_filters: [String] = null
```

Comment:
Please update the resolver logic in a way that ignoring legacy flags (`is_*`) when image_filters is specified and is not null. Also don't forget to describe such behaviors fluently at the field description. :-)